index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <div class="toolbar logo">John Doe</div>
+  <nav class="toolbar"><img src="image/Union.png" class="imagcss" alt="dropdown menu"></nav></header>
+  <div class="headline">
+<h1 class="primary">Hey There.
+  I’m Ghilain</h1>
+  <h2 class="support">I’am a Software Developer</h2>
+  <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.</p>
+    <ul class="social">
+      <li><a href="https://github.com/Ghilain"><img src="image/github.png" alt="Github logo"></a></li>
+      <li><a href="https://rw.linkedin.com/in/ghilain-ishimwe-067a5b1b4"><img src="image/linkedin.png" alt="linkedin logo"></a></li>
+      <li><a href="https://www.microverse.org/"><img src="image/peace.png" alt="social logo"></a></li>
+      <li><a href="https://twitter.com/GhilainIshimwe"><img src="image/twitter.png" alt="twitter logo"></a></li>
+      <li><a href="https://www.microverse.org/"><img src="image/gmail.png" alt="social logo"></a></li>
+    </ul><img src="image/illustration-header-mobile.png" class="illustrate" alt="vector image">
```

Comment:
- [ ] OPTIONAL: This image is a part of the design, therefore I recommend uploading it from `css` as a background `img`. 
- [ ] OPTIONAL: I recommend aligning the image to the right as provided in Figma design: